src/docs/product/issues/issue-details/performance-issues/index.mdx

```diff
@@ -86,6 +87,10 @@ The following is a list of available performance issues:
     </tr>
 </table>
 
+<Note>
```

Comment:
Would be great if you could add a link to this page in the EA section: https://docs.sentry.io/product/accounts/early-adopter/